src/wallet/rpc/wallet.cpp

```diff
@@ -637,68 +655,136 @@ RPCHelpMan simulaterawtransaction()
 
         include_watchonly = options["include_watchonly"];
     }
-
```

Comment:
In bf9deb407a8d73ac3b645b34c9bd6d28729a809c "rpc/wallet: Add details and duplicate section for simulaterawtransaction"

nit: Unrelated whitespace change